server/src/main/java/org/apache/calcite/avatica/server/HttpServer.java

```diff
@@ -850,6 +875,9 @@ protected SslContextFactory.Server buildSSLContextFactory() {
         sslFactory.setKeyStorePassword(keystorePassword);
         sslFactory.setTrustStorePath(truststore.getAbsolutePath());
         sslFactory.setTrustStorePassword(truststorePassword);
+        if (keystoreType != null && !keystoreType.equals(DEFAULT_KEYSTORE_TYPE)) {
+          sslFactory.setKeyStoreType(keystoreType);
```

Comment:
Could you add a comment here as to why this is important? Was there some use-case which can accept a keystore in a different type?

Reply:
Jetty supports other key-store formats as well, for example BCFKS key format. BCFKS is FIPS compatible but the default JKS is not.
This way we will be able configure the key-store format from phoenix-queryserver.

Reply:
thanks for the explanation.